### ISSUE TITLE
The animation parameter is not changing through Visual Scripting in Orchestrator.

### ISSUE THREAD

**Issue by arturbier** [CraterCrash/godot-orchestrator#819]
### Describe the bug

When using Visual Scripting to change an animation parameter through the node Set Parameters/conditions/isopen, the animation does not respond, and the parameter is not updated, despite correct setup of the nodes and parameters in the Animator.

### Expected behavior

The animation should start playing based on the changed isopen parameter through Visual Scripting.

### Actual behavior

The isopen parameter does not change, and the animation does not start.
![Снимок экрана 2024-10-07 в 15 35 15](https://github.com/user-attachments/assets/164f48b4-bbfc-4db8-86ae-ac222aafd1b9)

### How to Reproduce?

1.	Create a new project in Godot using the latest stable version.
	2.	Add an AnimationTree or AnimationPlayer node.
	3.	Set up an animation with a parameter (e.g., isopen) to control the animation states.
	4.	Enable Visual Scripting.
	5.	Create a graph with a Ready Event and connect it to the Set Parameters/conditions/isopen node.
	6.	Ensure that the parameter is passed correctly (e.g., set true for a boolean variable).
	7.	Run the project.

### Godot full version

v4.3 stable

### Orchestrator version

v2.2

**Comment by Naros** [comment]
Hi @arturbier, does it work if you use the animation node combined with using the `set` method, setting `parameters/conditions/isopen` with a value of `true`?

**Comment by arturbier** [comment]
> Hi @arturbier, does it work if you use the animation node combined with using the `set` method, setting `parameters/conditions/isopen` with a value of `true`?

No, none of the parameters work through visual scripting, for example, in “AnimationMixer - Active - True” it also doesn’t work. However, everything works fine through a regular script.

**Comment by Naros** [comment]
I recently had a discussion with another user and this does appear to work on the latest 2.3 builds. Can you please take a look and report back if you are able. If we don't hear back in 10 days, this issue will be automatically closed.
